docs/reference/metrics/aggregation_counter.md

```diff
@@ -0,0 +1,8 @@
+::: asab.metrics.metrics.AggregationCounter
+    handler: python
```

Comment:
These options can be set globally in `mkdocs.yml`. But we can keep them here for now.